tsfc/fiatinterface.py

```diff
@@ -178,6 +173,15 @@ def _(element, vector_is_mixed):
         # Real element is just DG0
         cell = element.cell()
         return create_element(ufl.FiniteElement("DG", cell, 0), vector_is_mixed)
+    if element.family() == "Quadrature":
+        # Sneaky import from FFC
+        from ffc.quadratureelement import QuadratureElement
+        ffc_element = QuadratureElement(element)
+
+        def tabulate(self, order, points, entity):
+            return QuadratureElement.tabulate(self, order, points)
+        ffc_element.tabulate = types.MethodType(tabulate, ffc_element)
+        return ffc_element
```

Comment:
> FFC elements are "FIAT elements" not-implementing the full FIAT element interface.

The `FiniteElement`/`CiarletElement` split somewhat mitigates this problem.

> how the suggestion fits an eventual FInAT transition and FIAT retirement.

It is mostly orthogonal, though any transition is easier with less chaos.

I will file proposal issue.